src/renderer/services/file-management-system/index.ts

```diff
@@ -233,8 +235,8 @@ export default class FileManagementSystem {
             result: [
               {
                 fileId,
-                fileName: path.basename(localPath),
-                readPath: localPath,
+                fileName: path.basename(metadata.file.originalPath), 
```

Comment:
is `originalPath` the path to the file on the user's system that FUA is uploading?  If so, the file can be given a different name in FMS using the `fileName` arg or the `/register` endpoint.  I would stick to basing this `fileName` key on `readPath` if that works.

Reply:
resolved in  [80a6911](https://github.com/aics-int/aics-file-upload-app/pull/116/commits/80a6911252d78d883e9b898ff9055f72500ae3ea) by just using the name from the file attires. Don't know the implications of this usage though.

Reply:
i think this is the safer thing to do.
Comparing a JSS job for a new upload to an older one would confirm.